maintainers/maintainer-list.nix

```diff
@@ -17374,6 +17374,12 @@
     github = "shymega";
     githubId = 1334592;
   };
+  siddarthkay = {
```

Comment:
Split.

Reply:
@AndersonTorres : could you please elaborate?

Reply:
Anderson means you should have two commits: one adding yourself to the maintainers list and another for adding the package. The commit message should be something like `maintainers: add siddarthkay`

Reply:
Thanks for explaining.